elide-example/elide-blog-example/README.md

```diff
@@ -15,8 +15,8 @@
 4. Launch the example webservice
 
         ~/elide $ mvn install
-        ~/elide $ cd elide-example/elide-persistence-mysql-example
-        ~/elide/elide-example/elide-persistence-mysql-example $ mvn exec:java -Dexec.mainClass="com.yahoo.elide.example.persistence.Main"
```

Comment:
Sections below will also need refactoring. There's no author or book model in this example